### ISSUE TITLE
storeState copy-paste bug in main.js

### ISSUE THREAD

**Issue by systemofapwne** [ioBroker/ioBroker.influxdb#37]
I'm currently trying to push custom data to my influxdb via storeState, yet, it fails, whenever I supply a single message with one state (I guess the same might be true for a single message with multiple state but not for multiple messages).
I tracked the problem down to main.js @ line 1244 and 1249

```
            id = aliasMap[msg.message.id] ? aliasMap[msg.message.id] : msg.message[i].id;
            pushValueIntoDB(id, msg.message.state[j]);
        }
    } else {
        adapter.log.debug('storeState: store 1 state for ' + msg.message.id);
        id = aliasMap[msg.message.id] ? aliasMap[msg.message.id] : msg.message[i].id;
```

should rather be

```
            id = aliasMap[msg.message.id] ? aliasMap[msg.message.id] : msg.message.id;
            pushValueIntoDB(id, msg.message.state[j]);
        }
    } else {
        adapter.log.debug('storeState: store 1 state for ' + msg.message.id);
        id = aliasMap[msg.message.id] ? aliasMap[msg.message.id] : msg.message.id;
```

and looks like a simple copy & paste error from line 1238.

Cheers.

**Comment by Apollon77** [comment]
Is this also happening with GitHub version?

**Comment by systemofapwne** [comment]
I just checked the code of it. I think it's fixed in the github version. Sorry, I forgot to try installing and testing it directly from here. Can be closed.

**Comment by Apollon77** [comment]
No leave open. My trigger to release the fix or at least check it.

**Comment by systemofapwne** [comment]
Well, judging on I don't see the bug in the code anymore, it is supposed to work. Will push the code soon to my iobroker in order to verify.

**Comment by systemofapwne** [comment]
And as expected, it works flawless. I have noticed, that this bug has been fixed in 6c8ffa9, 2 months ago.
Can be closed ;)
